sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -109,6 +118,14 @@ def preprocess_tod(obs_id, configs, overwrite=False, logger=None):
     for group in groups:
         logger.info(f"Beginning run for {obs_id}:{group}")
 
+        if qa_only:  # avoid loading data
+            # TODO: using the dets={group_by:group} argument didn't work in testing...
```

Comment:
needs fixes that are on this branch: https://github.com/simonsobs/sotodlib/pull/675